src/rules/font-family-name-quotes/__tests__/index.js

```diff
@@ -96,6 +98,21 @@ testRule(rule, {
   }, {
     code: "a { font-family: -apple-system, 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', sans-serif; }",
     message: messages.rejected("BlinkMacSystemFont"),
+  }, {
+    code: "a { font: italic 300 16px/30px Arial, serif; }",
+    message: messages.expected("Arial"),
+    line: 1,
+    column: 32,
+  }, {
```

Comment:
I'm not sure, what do browsers do with _unknown numerical font weights_, do they ignore the value, or round up/down to the nearest value?

Sounds like maybe a `font-weight-notation	-no-unknown` rule could be worth pursuing.